codegen/type_converter_test.go

```diff
@@ -1523,22 +1593,38 @@ func TestConverterMapStructWithSubFields2(t *testing.T) {
 
 	assert.NoError(t, err)
 	assertPrettyEqual(t, trim(`
-		if in.Three != nil {
-			out.Three = &structs.NestedBar{}
-			out.Three.One = string(in.Three.One)
-			if in.Four != nil && in.Four.Two != nil {
-				out.Three.One = *(in.Four.Two)
+		inOriginal := in; _ = inOriginal
```

Comment:
this looks wired, what does it do?

Reply:
In the current implementation this is kinda needed to support fieldMap overrides: within a function literal, only the "local" ```in``` and ```out``` are visible, but the fieldMap override could be taking in values from some other part of the original ```in```